packages/network/src/constants.ts

```diff
@@ -1,2 +1,22 @@
+import { IceServer } from './connection/webrtc/WebRtcConnection'
+
 export const COUNTER_UNSUBSCRIBE = -1
 export const DEFAULT_MAX_NEIGHBOR_COUNT = 4
+
+export const GOOGLE_STUN_SERVER: IceServer = {
+    url: 'stun:stun.l.google.com',
+    port: 19302
+}
+
+export const PRODUCTION_ICE_SERVERS: ReadonlyArray<IceServer> = Object.freeze([
```

Comment:
Could maybe rename this to `STREAMR_ICE_SERVERS`? Would be in sync with the `GOOGLE_... ` name.